daily-co/daily-demos
# feat: modified simple-demo with background images and gifs for Halloween

I modified our `simple-demo`, inspired by all of Mattie's celebratory demos, to showcase some Halloween background images and gifs. 

To test: 
1. Uncomment line 62 in `spooky.html`, and replace the placeholder `url` with your own Daily room URL. 
2. Replace line 15 with your own Giphy API key. 
3. Make sure you've followed the [instructions at the root of the daily-demos repo](https://github.com/daily-co/daily-demos)
4. `cd static-demos` 
5. `npm run start` or `npm run dev`
6. Then open your browser and go to `localhost:<port>/static-demos/spooky-demo/index.html`
